### ISSUE TITLE
Crash, no GUI and no tray icon in Kubuntu 23.04

### ISSUE THREAD

**Issue by mcarans** [slgobinath/SafeEyes#522]
**Describe the bug**
I have been using Safe Eyes for years. It is great and thanks for writing it. 

On upgrade to Kubuntu 23.04, it seems to crash on startup so no GUI appears, although the process itself keeps running. The tray icon does not appear (and I've checked it isn't hidden by KDE).

**To Reproduce**
1. Launch safeeyes with `safeeyes --debug &`
2. No UI or tray icon appears
3. `ps -ef` shows safeeyes is running
4. Launch safeeyes again with `safeeyes`
5. UI appears, but tstill no tray icon even if the tray icon plugin is activated

**Expected behavior**
GUI and tray icon appear on first startup

**Desktop (please complete the following information):**
 - OS: Kubuntu 23.04
 - Desktop Env KDE
 - Version 5.27.4

**Debug Log**
[safeeyes.log](https://github.com/slgobinath/SafeEyes/files/11378337/safeeyes.log)

**Comment by mcarans** [comment]
@slgobinath What are your plans regarding making a new release with latest Git?

**Comment by mcarans** [comment]
@hartwork @AdamPS I'm not sure who is maintaining this project but your names have come up as being closely involved.  Would it be possible to get a new release since the latest master branch fixes this bug?

**Comment by hartwork** [comment]
Hi @mcarans,

I'm only helping out with some bits, I'm not a maintainer.

After a quick look at the [debug log linked above](https://github.com/slgobinath/SafeEyes/files/11378337/safeeyes.log) I see multiple lines `Error in loading the plugin` which looks like a duplicate of #491 which was fixed by pull request #505. That would explain why Git `master` appears fixed to you. Before there is a new release, you can try downloading https://github.com/slgobinath/SafeEyes/commit/06d8f491c49613ad8dec3417be6b8ddc49abe46a.patch and using the `patch` command of GNU patch to fix your local installation.

@slgobinath what would it take to release SafeEyes 2.1.6 or 2.2.0 of latest `master` — how can I support you in this?

**Comment by AdamPS** [comment]
@slgobinath has granted me permission for some level of admin access here.

- I can definitely make commits here. I made a few at the start of the year, and would like to add a couple more before making a release, however I became busy.
- Probably I could make a release here, however I don't understand the full procedure. I guess we need to update translations and I don't know how. I only tested on my specific environment, and I may have made mistakes that would cause problems on a different environment.
- I don't have any other permission such as for the PPA, although maybe we are lucky and it has been set up to automatically detect releases here.

We should make a new issue to continue the conversation to create a new release.

**Comment by hartwork** [comment]
Hi @AdamPS,

> I [..] would like to add a couple more before making a release

are there notes on details somewhere? Could you summarize plans? Is this about release blockers or feature additions? Any particular issue tickets?


> * Probably I could make a release here, however I don't understand the full procedure. I guess we need to update translations and I don't know how. I only tested on my specific environment, and I may have made mistakes that would cause problems on a different environment.

Regarding translations, SafeEyes seems to be integrating with Weblate, see https://hosted.weblate.org/projects/safe-eyes/translations/ . Issue #245 is proof of that. I guess the question is how to get the Git repository back in sync with Weblate if they have new changes? (I would assume that Weblate keeps itself up to date with GitHub, not sure.)

Regarding releasing to PyPI, pushes to branch `release` combined with the GitHub Action at https://github.com/slgobinath/SafeEyes/blob/master/.github/workflows/release.yml may be all that's needed if still working. Here's what I would try in case of no release blockers:
- make a pull request to master bumping the version to 2.1.6
- if CI is green, merge to master
- checkout branch release, fast forward to latest remote master, push branch release, inspect CI logs

**Comment by hartwork** [comment]
@AdamPS PS: Regarding Weblate:

This may be handy to get an accurate live picture locally (but access is read-only):
```console
git remote add weblate-readonly https://hosted.weblate.org/git/safe-eyes/translations/
git remote update
```

Page https://docs.weblate.org/en/latest/admin/continuous.html probably has all the answers about our options.
But it's not clear to me what of that is set-up and working and how much more we can do without permissions in weblate.

**Comment by AdamPS** [comment]
Thanks for the comments, please start a new issue as I suggested😃. I won't have time for a few weeks at least. If anyone else can check the translation status that would help.

**Comment by mcarans** [comment]
I have made an issue for a new release: https://github.com/slgobinath/SafeEyes/issues/528
